test/StaticValidatorTest.php

```diff
@@ -154,7 +154,6 @@ public function testPassingNullWhenSettingPluginManagerResetsPluginManager()
 
     public function testExecuteValidWithParameters()
     {
-        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
```

Comment:
Oops, sorry. It seems you're right ^^. How about kept it with pass 3 parameters [false, 1, 10] ? 
I am away from pc now ;)

Warm regards,

Abdul Malik Ikhsan

Pada 8 Jul 2015, pukul 18.00, Alex Denvir <notifications@github.com> menulis:

> In test/StaticValidatorTest.php:
> 
> > @@ -154,7 +154,6 @@ public function testPassingNullWhenSettingPluginManagerResetsPluginManager()
> >  
> >      public function testExecuteValidWithParameters()
> >      {
> > -        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
> I agree, it probably should.
> But I'm afraid it doesn't:
> 
>  % php -a                                                                                                                                                                                                   code/zend-validator (master) two
> Interactive mode enabled
> 
> php > require_once "vendor/autoload.php";
> php > var_dump(\Zend\Validator\StaticValidator::execute(5, 'Between', ['min' => 1, 'max' => 10]));
> bool(true)
> php > var_dump(\Zend\Validator\StaticValidator::execute(0, 'Between', ['min' => 1, 'max' => 10]));
> bool(false)
> php > var_dump(\Zend\Validator\StaticValidator::execute(100, 'Between', ['min' => 1, 'max' => 10]));
> bool(false)
> php > var_dump(\Zend\Validator\StaticValidator::execute(5, 'Between', [1, 10]));
> bool(true)
> php > var_dump(\Zend\Validator\StaticValidator::execute(0, 'Between', [1, 10]));
> bool(true)
> php > var_dump(\Zend\Validator\StaticValidator::execute(100, 'Between', [1, 10]));
> bool(true)
> —
> Reply to this email directly or view it on GitHub.
>